get-config-state.js

```diff
@@ -40,6 +40,10 @@ function getConfigState(dirname, opts) {
         dc && NODE_ENV ?
             join(configFolder, NODE_ENV + '.' + dc.datacenter + '.json') :
             null,
+        // load ./config/secrets-NODE_ENV.json
+        NODE_ENV ? join(configFolder, 'secrets-' + NODE_ENV + '.json') : null,
+        // load ./config/secrets.json
```

Comment:
I think it makes sense for it to be /config/secret/*, the plan is to leverage symlinks. And that makes a single symlink